song/app.js

```diff
@@ -31,10 +31,53 @@ app.use(morgan("dev"));
 app.get("/ping", function (req, res, next) {
   res.json({ message: "pong" });
 });
-
+// 게시물 목록 조회
+app.get("/list", async (req, res, next) => {
+  await appDataSource.query(
+    `SELECT
+            users.id as userId,
+            users.profile_image as userProfileImage,
+            posts.user_id as postingId,
+            posts.image_url as postingImageUrl,
+            posts.content as postingContent
+            FROM users LEFT JOIN posts ON users.id = posts.user_id`,
```

Comment:
게시글 목록을 가져오는 SQL이므로 posts 테이블을 기준으로(FROM posts) 데이터를 가져와 보세요.
```suggestion
            FROM posts 
            JOIN users ON users.id = posts.user_id`,
```